modules/bitcoind.nix

```diff
@@ -182,7 +183,7 @@ in {
       };
       proxy = mkOption {
         type = types.nullOr types.str;
-        default = null;
+        default = if cfg.enforceTor then config.services.tor.client.socksListenAddress else null;
```

Comment:
I don't agree with linking the proxy setting with `cfg.enforceTor`. There are situations in which a user might want to use a Tor proxy but still not enforceTor, for example when he wants Bitcoind to communicate out over Tor, but accept incoming clearnet connections.

Reply:
This use case is still possible, as before.
This setting just automatically sets `proxy` when `enforceTor == true`. For the mentioned use case the user would just set the `proxy` manually and leave `enforceTor` disabled.

Reply:
Makes sense